porting/common-problems-run.rst

```diff
@@ -0,0 +1,89 @@
+Common problems when running Ubuntu Touch
+-----------------------------------------
+
+This page details problems commonly faced while following the :doc:`running-ut` page.
+
+Nothing shows on screen
+-----------------------
+
+If nothing is showing on screen even after adding udev rules to your port, it is likely that you have a problem with graphical applications crashing. See `Mir servers crash`_ for more information.
+
```

Comment:
servers -> server

Reply:
In this case, it is "servers", since unity-system-compositor, the mir tests, and unity8 itself can all run/contain a Mir server